.github/workflows/merlin.yml

```diff
@@ -397,13 +417,8 @@ jobs:
         uses: actions/download-artifact@v2
         with:
           name: merlin-transformer.${{ needs.create-version.outputs.version }}.tar
-      - name: Download k3d
-        run: |
-          curl --silent --fail https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | TAG=v5.4.1 bash
-      - name: Create Test Cluster
-        run: |
-          k3d registry create $LOCAL_REGISTRY --port $LOCAL_REGISTRY_PORT
-          k3d cluster create $K3D_CLUSTER --image rancher/k3s:v1.25.9-k3s1 --k3s-arg '--disable=traefik,metrics-server@server:*' --port 80:80@loadbalancer
+      - name: Check disk size
+        run: df -h
```

Comment:
This is for debugging purposes. I'm still trying to have e2e-test run successfully in GitHub Actions. The current issue is `Node k3d-merlin-cluster-server-0 status is now: NodeHasNoDiskPressure`